workflow/tests/ACCA_Examples/Bob_Ross_Residence.xml

```diff
@@ -0,0 +1,1483 @@
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>2.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>6.25</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallPrimaryBedroomNorth'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowPrimaryBathroomNorth'/>
+            <Area>10.0</Area>
+            <Orientation>north</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowPrimaryBathroomNorthExteriorShading'/>
+              <SummerShadingCoefficient>0.8</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowPrimaryBathroomNorthInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>2.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>4.5</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallPrimaryBathroomNorth'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='FrenchDoorFamilyRoomNorth'/>
+            <Area>35.0</Area>
+            <Orientation>north</Orientation>
+            <UFactor>0.6</UFactor>
+            <SHGC>0.39</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='FrenchDoorFamilyRoomNorthExteriorShading'/>
+              <SummerShadingCoefficient>1.0</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='FrenchDoorFamilyRoomNorthInteriorShading'/>
+              <SummerShadingCoefficient>1.0</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>3.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>10.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallFamilyRoomNorth'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowFamilyRoomEast'/>
+            <Area>26.7</Area>
+            <Orientation>east</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowFamilyRoomEastExteriorShading'/>
+              <SummerShadingCoefficient>1.0</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowFamilyRoomEastInteriorShading'/>
+              <SummerShadingCoefficient>0.93</SummerShadingCoefficient>
```

Comment:
Hmm, how did you get this? The Word doc says "Family Room, Row 4 (2P Clear), Drape Low", which is 0.34. And the house plan says 50% drawn, so I get (1+0.34)/2 = 0.67.

Reply:
Yes I'm confused here too, the word doc said Drape Low but the pdf said dark, loose weave, which is Drape High (in 3D-4, High = loose weave, dark color; Average = medium weave, medium color; Low = tight weave, white or reflective color, so High is what I used here), I think that's where our calculations deviates.